tests/conftest.py

```diff
@@ -104,18 +103,18 @@ def _import_and_add_jax(xp_available_backends: dict[str, types.ModuleType]) -> N
 def cosmo() -> Cosmology:
     class MockCosmology:
         @property
-        def omega_m(self) -> float:
+        def Omega_m0(self) -> float:  # noqa: N802
             """Matter density parameter at redshift 0."""
             return 0.3
 
         @property
-        def rho_c(self) -> float:
+        def critical_density0(self) -> float:
             """Critical density at redshift 0 in Msol Mpc-3."""
             return 3e4
 
-        def ef(self, z: NDArray[np.float64]) -> NDArray[np.float64]:
+        def H_over_H0(self, z: NDArray[np.float64]) -> NDArray[np.float64]:  # noqa: N802
             """Standardised Hubble function :math:`E(z) = H(z)/H_0`."""
-            return (self.omega_m * (1 + z) ** 3 + 1 - self.omega_m) ** 0.5
+            return (self.Omega_m0 * (1 + z) ** 3 + 1 - self.Omega_m0) ** 0.5
 
         def xm(
```

Comment:
I've left `xm` as it is as it's not being used outside of `MockCosmology` currently